CHANGELOG.md

```diff
@@ -8,13 +8,22 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [Unreleased]
 
 ### Added
-- Introduce latency observation for http endpoint (#17).
-- Support `roles.httpd` integration (#15).
 
 ### Fixed
 
 ### Changed
 
+## 0.2.0 - 2024-10-02
+
+The release introduces the integration with `httpd` role and latency observation for http
+endpoint. Now it is possible to reuse server's address from `httpd` role config for
+`metrics-export-role` configuration. The release contains documentation about these changes.
```

Comment:
It's usually better to include a link to documentation than to just write about it.

Up to you.
```suggestion
endpoint. Now it is possible to reuse server's [address from `httpd` role config for](link)
`metrics-export-role` configuration.
```

Reply:
Added reference to the README's documentation.